vars/copyFiles.groovy

```diff
@@ -0,0 +1,16 @@
+def call(sourceDirectory, destinationDirectory, files=null) {
+   def copyCommand = "robocopy \"$sourceDirectory\" \"$destinationDirectory\""
+   def commandSwitches = "/NFL /NDL /NJH /NJS /nc /ns /np"
```

Comment:
Can we document/rename the variable to better explain what all of these switches do? For someone not intimately familiar with robocopy, this is confusing.